docs/usage/faq.md

```diff
@@ -0,0 +1,272 @@
+# Frequently asked questions and troubleshooting
+
+## Recommended auxiliary files
+
+Some tools may require or recommend additional files to the reference sequence files - such as taxonomy files - to execute.
+
+We provide a list of required or recommended files, and which pipeline parameters to give them to here:
+
+- bracken
+  - taxonomy name dump file (`--namesdmp`)
+  - taxonomy nodes dump file (`--nodesdmp`)
```

Comment:
this is an option for all tools. It is slightly confusing to repeat this in the docs this way.

Reply:
yeah, but only one option for multiple tools.
that's slightly confusing I believe